benchmarker.py

```diff
@@ -102,7 +115,7 @@ def save_to_html(results):
 	heading = "<th></th>" + "".join([ '<th style="font-style:bold;text-align:center;padding:5px 20px;">' + next + "</th>" for next in items])
 	html += "<tr>" + heading + "</tr>"
 	for i in range(len(languages)):
-		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str(round(next, 3)) + " s" + "</td>" for next in results[i]])
+		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str("{:.3f}".format(round(next, 3))) + " s" + "</td>" for next in results[i]])
```

Comment:
I see! Then you can remove the round() call?
On Jun 17, 2016 15:21, "Deniz Kennedy" <notifications@github.com> wrote:

> In benchmarker.py
> <https://github.com/naslundx/language-benchmarker/pull/22#discussion_r67508680>
> :
>
> > @@ -102,7 +115,7 @@ def save_to_html(results):
> >  	heading = "<th></th>" + "".join([ '<th style="font-style:bold;text-align:center;padding:5px 20px;">' + next + "</th>" for next in items])
> >  	html += "<tr>" + heading + "</tr>"
> >  	for i in range(len(languages)):
> > -		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str(round(next, 3)) + " s" + "</td>" for next in results[i]])
> > +		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str("{:.3f}".format(round(next, 3))) + " s" + "</td>" for next in results[i]])
>
> Sometimes, it finishes exactly like, for example, 0.06 so instead of
> printing out 0.060 s, it prints out 0.06 s .It's not big deal, but just
> aligning everything :) , what do you think?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/naslundx/language-benchmarker/pull/22/files/d22dcd199d3341456b99529798e7d8f8961f24e3#r67508680>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACFvhDBo0oCbx9AtBpYRc9TcXDtrnnXOks5qMp9igaJpZM4I31jF>
> .
>